src/components/button/Button.js

```diff
@@ -215,7 +215,7 @@ export default class ButtonComponent extends BaseComponent {
           event.preventDefault();
           event.stopPropagation();
           this.emit('submitButton', {
-            state: this.component.state || 'submitted'
+            state: this.component.action === 'submit' ? 'submitted' : 'draft'
```

Comment:
This change is incorrect. The default state should be 'submitted' unless they specifically set it to draft. Your testing form may not have been configured properly. Set it to "Save in State" and State of "draft" to test.